pykickstart/commands/rootpw.py

```diff
@@ -107,7 +106,8 @@ def _getParser(self):
                         version=F8, help="""
                         If this is present, the root account is locked by
                         default. That is, the root user will not be able to
-                        login from the console.""")
+                        login from the console. When this option is present
+                        the <password> argument is not required.""")
```

Comment:
@Frodox I think what you're trying to tell me got eaten by markdown. Do you mean to use backticks to escape the "<password>" string so that it is rendered differently than the rest when RTD builds the HTML ?